ConvivaIntegration/components/conviva/ConvivaAnalyticsTask.brs

```diff
@@ -297,3 +313,20 @@ end sub
 sub debugLog(message as String)
   if m.DEBUG then ?message
 end sub
+
+function getAd(mediaId)
+  adBreaks = m.top.player.callFunc(m.top.player.BitmovinFunctions.AD_LIST)
+  for each adBreak in adBreaks
+    for each ad in adBreak.ads
+      if ad.id = mediaId then return ad
+    end for
+  end for
+
+  return "NA"
```

Comment:
Is this return value some kind of magic value? If yes, please add a comment with additional info, If no, just return `invalid`.